### PR TITLE
Motor numbering 173

### LINKED ISSUE #173: Fix wheel motor numbering
In `robot/rover/MobilePlatform/commands.h`, remove the need for the todo seen here : https://github.com/space-concordia-robotics/robotics-prototype/commit/46a28fa0168fcf09380338e88b9e3291eba176f2

- [ ]  make sure motor numbering makes sense in both closed and open loop (1 to 6 not 0 to 5)
- [ ] make sure motor order is correct (1 to 6 actually turns motors 1 to 6)

### PR DESCRIPTION
**Pull request by fauziame**
# Assignee Section

## Description

Updated motor numbering in firmware to be consistent with the general nomenclature.

## Local Testing steps

1. Upload `MobilePlatform.ino` to a teensy

closes #173

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [ ] Local Test Performed Successfully
- [ ] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

For Pull Requests that do not include code changes, it is not required to perform the tests above.

### CODE REVIEW

**Comment by fauziame** [comment]
Duplicate Pull request!

File: robot/rover/MobilePlatform/commands.h [hunk 1]
```diff
@@ -550,46 +550,43 @@ void Commands::controlWheelMotors(String cmd) {
...
             }
-            if (motorNumber == 5){
+            else if (motorNumber == 6){
                 LB.calcCurrentVelocity();
                 LB.setVelocity(dir , abs(motorSpeed), LB.getCurrentVelocity());
                 println("ASTRO " + String(LB.motorName) + String("'s desired speed: ") + String(LB.desiredVelocity) + String(" PWM "));
-
+            }
+            else {
+                println("ASTRO invalid motor  number")
```
Comment (MartensCedric) [inline robot/rover/MobilePlatform/commands.h]: double space

File: robot/rover/MobilePlatform/commands.h [hunk 2]
```diff
@@ -550,46 +550,43 @@ void Commands::controlWheelMotors(String cmd) {
...
-            if (motorNumber == 4){
+            else if (motorNumber == 5){
                 LM.calcCurrentVelocity();
                 LM.setVelocity(dir , abs(motorSpeed), LM.getCurrentVelocity());
                 println("ASTRO " + String(LM.motorName) + String("'s desired speed: ") + String(LM.desiredVelocity) + String(" PWM "));
-
             }
-            if (motorNumber == 5){
+            else if (motorNumber == 6){
                 LB.calcCurrentVelocity();
```
Comment (MartensCedric) [inline robot/rover/MobilePlatform/commands.h]: I suggest using a switch case since it's a bit more elegant.

**Comment by MartensCedric** [comment]
@fauziame can you also fill the PR description?

**Pull request merged**